### ISSUE TITLE
A.3.3

### ISSUE THREAD

**Issue by jensscheerlinck** [inspire-eu-validation/ats-discovery-service#23]
_This issue has been extracted from the issue list on:https://ies-svn.jrc.ec.europa.eu/issues/2685_
# Comment

The question is what is the difference between this test and A.3.2?

**Comment by jensscheerlinck** [comment]
# Proposed Resolution

Remove test case A.3.2
Not based on technical guideline and is covered by test A.3.3

**Comment by PeterParslow** [comment]
Agreed: A.3.2. is more generic - it should reference requirement 9, but that also is generic, implemented by requirement 10 (this test case) and 11 (A.3.4 and so on).
